benchexec/tools/afltc.py

```diff
@@ -54,7 +57,20 @@ def cmdline(self, executable, options, task, rlimits):
                 "The 'data_model' option must be either 'ILP32' or 'LP64'."
             )
 
-        return [executable, task.single_input_file, data_model_option]
+        command_line = [executable, task.single_input_file, data_model_option]
+
+        # The first version(s) (all denoted as "0.1.0") did not support property files
+        version = self.version(executable)
```

Comment:
This would mean that we need to cache the call to get the version, otherwise we execute the tool many times. This is easy with `@functools.cache`.